argo-lite/src/components/FloatingCards.jsx

```diff
@@ -34,7 +35,31 @@ class RenderOptionsCard extends React.Component {
 
 @observer
 class FloatingCards extends React.Component {
+  constructor (props) {
+    super (props);
+  this.state = {
+    isRenderOptionsCardHidden: false 
```

Comment:
Is it possible to move this to use mobx stores rather than local states? Many such dialog options are in preference store if I remember correctly.

Reply:
This helps with situations where we want to programmatically hide the side panel later on (eg. if we reintroduce the search bar and need to hide the panel when search is going on)